projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-actions-docs/table-actions-docs.component.html

```diff
@@ -15,6 +15,14 @@ <h2 class="go-heading-2 go-heading--no-wrap">Implementing Table Actions</h2>
       <p class="go-body-copy">
         For the table actions to appear, you must toggle them on by using <code class="code-block--inline">showTableActions</code>
       </p>
+      <p class="go-body-copy">
+        Also, the size of the button icons can be controlled via the binding <code class="code-block--inline">buttonSize</code>.
+        The design guidelines suggest using <code class="code-block--inline">buttonSize="medium"</code>, although you can choose 
+        from <code class="code-block--inline">small</code>, <code class="code-block--inline">medium</code> or <code class="code-block--inline">large</code>.
+        For more details, you can refer to the section Button Size, under Icon Button component.
```

Comment:
I appreciate your thoroughness here, but we probably don't need to document the other sizes. The specification from UX is that we should only be using `medium` for the icons here. I like that you refer the reader to the icon button docs, what do you think about providing a link to that?

```
For more details, you can refer to the Button Size section on the 
<a routerLink="/ui-kit/icon-button">Icon Button documentation</a>.
```

Reply:
done and dusted !